README.md

```diff
@@ -51,6 +51,10 @@ sudo apt install qttools5-dev qttools5-dev-tools qtbase5-dev qtchooser qt5-qmake
 ## Building from sources
 
 ```sh
+# If you have installed radare2 from sources in /usr/local/ you might need to do this:
+export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig
+export LD_LIBRARY_PATH=/usr/local/lib
```

Comment:
Thats a linux specific env var. on mac its DYLD_... but ideally rpath should be used at compile time to avoud this confusion in a specific binary instead of shell-wide.

also. There's ld.so.conf to handle that at system level

Reply:
Understood